### PR TITLE
TDH-1004 -: Load the files based on prop

### PR DESCRIPTION
**Pull request by panwaranuj01**
### What do you want to achieve?
-

### PR Checklist

- [x] I have tested it locally and all functionalities are working fine.
- [x] I have compared it with mocks and all design elements are the same.
- [ ] I have tested it in IE Browser.

### How was the code tested?

-

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

### Screenshot
- 

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

File: webplugin/js/app/mck-app.js [hunk 1]
```diff
@@ -311,6 +317,82 @@ function ApplozicSidebox() {
...
+                        async (file) => {
+                            promises.push(loadResourceAsync(file));
+                        }
+                    );
+                }
+            }
+
+            if (options.voiceNote) {
+                promises.push(
+                    loadResourceAsync(THIRD_PARTY_SCRIPTS.recorder.js)
```
Comment (devashishmamgain) [inline webplugin/js/app/mck-app.js:379]: if recorder and frvoice are both required together then better to have a single combined minified file for them

**Review by devashishmamgain — COMMENTED** [review]
@panwaranuj01 does it work for case where the pre chat lead collection script is required for telephone numbers. do we wait until the script is loaded before showing the widget? or widget gets shown and the user is free to click on the chat icon

**Comment by panwaranuj01** [comment]
> @panwaranuj01 does it work for case where the pre chat lead collection script is required for telephone numbers. do we wait until the script is loaded before showing the widget? or widget gets shown and the user is free to click on the chat icon

we are waiting till file not loaded fully.

File: webplugin/js/app/mck-app.js (hunk 1 above)
Comment (panwaranuj01) [reply webplugin/js/app/mck-app.js:379]: That is good suggestion it will not make the extra call

Comment (panwaranuj01) [reply webplugin/js/app/mck-app.js]: done

**Review by devashishmamgain — DISMISSED** [review]

**Review by devashishmamgain — APPROVED** [review]

**Review by devashishmamgain — APPROVED** [review]

**Pull request merged**